src/components/Admin/Settings/Mail/index.js

```diff
@@ -0,0 +1,421 @@
+                margin="normal"
+                value={frontendUrl || ''}
+                name="frontendUrl"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '36%' }}
+              />
+              <TextField
+                label="Sendgrid Token"
+                placeholder="Sendgrid Token"
+                margin="normal"
+                value={sendgridToken || ''}
+                fullWidth={true}
+                name="sendgridToken"
+                onChange={this.handleChange}
+                disabled={type !== 'Sendgrid'}
+                variant="outlined"
+                style={{ width: '63%' }}
+              />
+            </FlexContainer>
+            <FlexContainer>
+              <TextField
+                label="Trust Self-Signed Certificate"
+                placeholder="Trust Self-Signed Certificate"
+                margin="normal"
+                value={trustselfsignedcerts || ''}
+                name="trustselfsignedcerts"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '32%' }}
+              />
+              <TextField
+                label="Encryption"
+                placeholder="Encryption"
+                margin="normal"
+                value={encryption || ''}
+                name="encryption"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '32%' }}
+              />
+              <TextField
+                label="SMTP Port"
+                placeholder="SMTP Port"
+                margin="normal"
+                value={port || ''}
+                name="port"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '32%' }}
+              />
+            </FlexContainer>
+            <FlexContainer>
+              <TextField
+                label="Username"
+                placeholder="Username"
+                margin="normal"
+                value={username || ''}
+                name="username"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '49%' }}
+              />
+              <TextField
+                label="Password"
+                placeholder="Password"
+                margin="normal"
+                value={password || ''}
+                name="password"
+                onChange={this.handleChange}
+                variant="outlined"
+                style={{ width: '49%' }}
+              />
+            </FlexContainer>
+            <ExpansionPanel style={{ margin: '1rem 0' }}>
+              <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+                <Typography>Send Test Email</Typography>
+              </ExpansionPanelSummary>
+              <ExpansionPanelDetails>
+                <TextField
+                  label="Receiver Email"
```

Comment:
Send email doesn't send any mail as of now.

Reply:
I tried looking into it, this would take time, to find where the problem lies